substrate/frame/revive/Cargo.toml

```diff
@@ -23,7 +23,7 @@ derive_more = { workspace = true }
 environmental = { workspace = true }
 ethabi = { workspace = true }
 ethereum-types = { workspace = true, features = ["codec", "rlp", "serialize"] }
-hex = { workspace = true }
+hex-literal = { workspace = true }
```

Comment:
Could you please use `array-bytes` as used in the rest of the code base?

Reply:
Both seem to be widely used in the code base.  Should we transition away from `hex-literal`?
```
alexander:~/Developer/polkadot-sdk% cargo tree -i hex-literal --depth 1 | wc -l
      22
alexander:~/Developer/polkadot-sdk% cargo tree -i array-bytes --depth 1 | wc -l
      34
```

Reply:
At some point we tried to push for `array-bytes`, but all the other are coming back over time :P

Reply:
Just looked into `array-bytes`. It is not a replacement since it does the conversion at runtime. But we often need it at compile time as literal. Or do you mean the `hex` crate? I removed the direct dependency but we are still indirectly depending on it via `ethereum-types`.  We will move away eventually to the newer `alloy` crates.